### PR TITLE
Add Xfer Records OP-1 Drum Utility 1.02

### PR DESCRIPTION
**Pull request by danielbayley**
Closes https://github.com/caskroom/homebrew-cask/pull/40173.

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-drivers/pulls
[closed issues]: https://github.com/caskroom/homebrew-drivers/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/op-1-drum-utility.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'op-1-drum-utility' do
```
Comment (commitay) [inline Casks/op-1-drum-utility.rb]: Please prepend the vendor name to the token. [naming-driver-casks](https://github.com/caskroom/homebrew-drivers/blob/master/CONTRIBUTING.md#naming-driver-casks)

**Comment by commitay** [comment]
@danielbayley Is the `preflight` needed or can we do this?

```
  app "OP1DrumUtilityOSX_#{version.no_dots}/OP-1 Drum Utility.app"
  vst_plugin "#{appdir}/OP-1 Drum Utility.app/Contents/Resources/Op1DrumUtil.vst"
```

**Comment by danielbayley** [comment]
@commitay That breaks because the .app version is a wrapper around [needs] the vst internally, so it has to be copied…

**Comment by commitay** [comment]
Will installing both work? We shouldn't assume which one a user would prefer.

```
If you prefer to run OP-1 Drum Utility as a VST:
• In the Finder, right-click (ctrl-click) on the OP-1 Drum Utility application icon and select “Show Package Contents”
• Navigate to /Contents/Resources/
• Copy (do not move, or the app will no longer work) the file “Op1DrumUtil.vst” file to /Library/Audio/Plug-Ins/VST/
```
```
If you prefer to run OP-1 Drum Utility as an AudioUnit:
• In the Finder, right-click (ctrl-click) on the OP-1 Drum Utility application icon and select “Show Package Contents”
• Navigate to /Contents/Resources/
• Copy (do not move, or the app will no longer work) the “Op1DrumUtil.vst” file to /Library/Audio/Plug-Ins/Components/
• rename this copy you have just put in /Components/ to “Op1DrumUtil.component” .
```

**Comment by danielbayley** [comment]
@commitay Ah so there is actually a third way… It should probably cover all of them then.

**Pull request merged**